queries.yaml

```diff
@@ -168,3 +168,7 @@ post_process:
     source_layer: landuse
     label_property_name: label_position
     label_property_value: "yes"
+  TileStache.Goodies.VecTiles.transform.generate_label_features:
```

Comment:
Unfortunately, `post_process` is currently a `dict`, so we can't have the same key multiple times. There's no good reason for it to be that way, it was just the easiest way at the beginning. I'm not sure whether this would require a code change - needs to be looked into.